ui/app/AppLayouts/Chat/components/ProfilePopup.qml

```diff
@@ -109,7 +109,7 @@ ModalPopup {
 
     StyledText {
       id: valueEnsName
-      text: "@emily.stateofus.eth"
+      text: userName
```

Comment:
This element should be shown/hidden depending if the user has a ens username. I think userName will return the alias if the ENS is not available.